### PR TITLE
No longer continually vetting masquerades

### PR DESCRIPTION
**Pull request by oxtoacart**
Sorry the diff on util_test is so big, I just added all the masquerades because it was failing with its initial masquerades.  Eventually, we should just have one place where we keep the list of masquerades (see #3587).

### CODE REVIEW

File: src/github.com/getlantern/fronted/direct.go [hunk 1]
```diff
@@ -76,26 +77,28 @@ func (d *direct) vet(initial map[string][]*Masquerade) {
...
-		go d.vetSome()
+func (d *direct) vetInitial() {
+	log.Debugf("Vetting %d initial candidates in parallel", NumberToVetInitially)
+	for i := 0; i < NumberToVetInitially; i++ {
+		go d.vetOne()
 	}
 }
 
-func (d *direct) vetSome() {
+func (d *direct) vetOne() {
```
Comment (oxtoacart) [inline src/github.com/getlantern/fronted/direct.go]: This goroutine now terminates once it has successfully verified one masquerade.  So, when we reconfigure, we don't end up piling up a bunch of goroutines.  Also, we don't do a bunch of work vetting masquerades that we may never use.

**Comment by fffw** [comment]
LGTM. Travis failed because of an unresolved dependency, all relevant test were passed locally.

**Comment by fffw** [comment]
Plus: The unnecessary dependency of Travis is removed in valencia. Manually running Lantern for 1+ hours on Mac, memory keeps at 30MB.

**Comment by oxtoacart** [comment]
@fffw Thanks for reviewing!

**Comment by myleshorton** [comment]
Awesome Ox and thanks for testing Merlin! I'll get this into a new build
ASAP.

**Pull request merged**